microsoft-edge/webview2/index.md

```diff
@@ -20,6 +20,32 @@ With WebView2, you can embed web code in different parts of your native app, or
 To start building a WebView2 app, see [Get started with WebView2](get-started/get-started.md).
 
 
+<!-- ====================================================================== -->
+## Full diagram of WebView2
+
+Developing a WebView2 app has the following high-level areas:
+*  Dev machine
+*  Distribute the app and Runtime
+*  User machine
+*  Resources
+
+![Full diagram of WebView2](./index-images/full-diagram.png)
```

Comment:
@nishitha-burman @peiche-jessica - pls review diagram:

.png link:
https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/wv2-intro-diagram/microsoft-edge/webview2/index-images/full-diagram.png

gh rendered:
https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/wv2-intro-diagram/microsoft-edge/webview2/index.md#full-diagram-of-webview2

eg how to describe relation of right-side blocks:

Native-side code | Web-side code

Native control instances | WebView2 control instances

Native-side code includes:
*  Code that's related to WebView2
*  Code that's not related to WebView2